src/MasterCrud.php

```diff
@@ -205,7 +205,7 @@ public function initCrud(array $defs, View $view = null)
         $crud->setModel($this->model);
 
         if (isset($crud->table->columns[$this->model->title_field])) {
-            $crud->addDecorator($this->model->title_field, [Table\Column\Link::class, [], [$this->model->table . '_id' => 'id']]);
+            $crud->addDecorator($this->model->title_field, [Table\Column\Link::class, [], [$this->model->table . '_id' => $this->model->idField]]);
```

Comment:
`$this->model->table . '_id'` could be probably replaced with already build name from a reference object